### PR TITLE
[Java] Build documentation for Java

### LINKED ISSUE #423: [Java] Generate documentation for Java bindings
For example, OpenCV: https://docs.opencv.org/4.x/javadoc/index.html

### PR DESCRIPTION
**Pull request by dkurt**
resolves #423

Generated docs: https://openvino-java.readthedocs.io/

@likholat, please review. Also, I will add you as a maintainer to https://readthedocs.org/projects/openvino-java/

### CODE REVIEW

File: modules/java_api/cpp/openvino_java.hpp [hunk 1]
```diff
@@ -163,7 +163,7 @@ extern "C"
     JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_GetPrePostProcessor(JNIEnv *, jobject, jlong);
     JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_Input(JNIEnv *, jobject, jlong);
     JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_Output(JNIEnv *, jobject, jlong);
-    JNIEXPORT void JNICALL Java_org_intel_openvino_PrePostProcessor_Build(JNIEnv *, jobject, jlong);
+    JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_Build(JNIEnv *, jobject, jlong);
```
Comment (dkurt) [inline modules/java_api/cpp/openvino_java.hpp:166]: Please take attention to this change. According to https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/preprocess/pre_post_process.hpp, `build()` method should return a reference to `Model`.

File: modules/java_api/org/intel/openvino/PrePostProcessor.java [hunk 2]
```diff
@@ -13,16 +22,35 @@ public PrePostProcessor(Model model) {
...
     }
 
-    public void build() {
-        Build(nativeObj);
+    /**
+     * Adds pre/post-processing operations to function passed in constructor
+     *
+     * @return Function with added pre/post-processing operations
+     */
+    public Model build() {
```
Comment (likholat) [inline modules/java_api/org/intel/openvino/PrePostProcessor.java:52]: then native method `Build` can't be `void`

Comment (dkurt) [reply modules/java_api/org/intel/openvino/PrePostProcessor.java:52]: Thanks! Done

**Comment by ilya-lavrenov** [comment]
Please, let's know if it's ready to merge

**Review by likholat — APPROVED** [review]

**Comment by likholat** [comment]
@ilya-lavrenov, it's ready to merge

**Pull request merged**